### PR TITLE
.helm: add cronjob to backup database

### PR DESCRIPTION
**Pull request by BacLuc**
How to create and upload the backup was inspired from here: https://github.com/itbm/postgresql-backup-s3/blob/master/backup.sh

How to encrypt/decrypt de backup was inspired by here: https://www.cyberciti.biz/tips/linux-how-to-encrypt-and-decrypt-files-with-a-password.html

To decrypt the backup:
`gpg --passphrase=${ENCRYPTION_KEY} --batch -d backup.sql.gz.gpg > backup.sql.gz; gunzip backup.sql.gz`

Issue: #3478

### CODE REVIEW

File: .helm/deploy-to-cluster.sh [hunk 1]
```diff
@@ -5,7 +5,9 @@ set -e
 SCRIPT_DIR=$(realpath "$(dirname "$0")")
 REPO_DIR=$(realpath "$SCRIPT_DIR"/..)
 
+set -f
```
Comment (usu) [inline .helm/deploy-to-cluster.sh:8]: What is this for?

**Review by usu — APPROVED** [review]

File: .helm/deploy-to-cluster.sh (hunk 1 above)
Comment (BacLuc) [reply .helm/deploy-to-cluster.sh:8]: for disabling globbing for cron expressions (* * * * *) is globbed to (api  cache  common  CONTRIBUTING.md  create-ecamp3-test-database.sh  docker-compose.yml  e2e  frontend  LICENSE  print  README.md  renovate.json  translation api  cache  common  CONTRIBUTING.md  create-ecamp3-test-database.sh  docker-compose.yml  e2e  frontend  LICENSE  print  README.md  renovate.json  translation...)
But the glob expressions don't work anyway, currently we can use @weekly or @midnight

**Review by simfeld — APPROVED** [review]

**Comment by carlobeltrame** [comment]
Merge together with ecamp/ecamp-site#20 (after specifying the datacenter country if possible)

**Comment by BacLuc** [comment]
superseded by https://github.com/ecamp/ecamp3/pull/3640

**Pull request closed without merging**